generated/index.d.ts

```diff
@@ -165,7 +165,8 @@ type GPUFeatureName =
     | "float32-blendable"
     | "clip-distances"
     | "dual-source-blending"
-    | "subgroups";
+    | "subgroups"
+    | "core-features-and-limits";
```

Comment:
Only need to add it to the dist/ file. This one stays "clean" with the generator so we can see what changes when we roll gpuweb.

```suggestion
    | "subgroups";
```